bfh/transformations.py

```diff
@@ -365,9 +365,14 @@ class DateToIsoString(Transformation):
     Turn a datetime into an ISO 8601 formatted string.
 
     """
+    null_types = (None,)
+
     def function(self, source, *call_args):  # source ignored
         try:
-            return call_args[0].isoformat()
+            value = call_args[0]
+            if value in self.null_types:
+                return value
+            return value.isoformat()
```

Comment:
It seems me that only this line should be inside the `try` call. The new code should be added before the `try` call, right?